mockall_derive/src/mock_function.rs

```diff
@@ -986,15 +986,22 @@ impl<'a> ToTokens for Common<'a> {
 
             impl #ig Drop for Common #tg #wc {
                 fn drop(&mut self) {
-                    if !::std::thread::panicking() && !self.times.is_satisfied()
-                    {
-                        let desc = std::format!(
-                            "{}", self.matcher.lock().unwrap());
-                        panic!("{}: Expectation({}) called {} time(s) which is fewer than expected {}",
-                               #funcname,
-                               desc,
-                               self.times.count(),
-                               self.times.minimum());
+                    if !::std::thread::panicking() {
+                        match self.times.is_satisfied(); {
```

Comment:
will remove ; in next commit

Why cargo build doesn't catch there errors?
How should I compile?

Reply:
Well, if you just did `cargo build`, that would build the proc macros and the small amount of code in the mockall crate, but it wouldn't evaluate any of the proc macros, anywhere.  You should do `cargo test` before pushing.